.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
-  - "node"
-  - "4" # Fedora
+  - "0.10" # EL6, Trusty, Jessie
```

Comment:
For Debian it would be possible to use nodejs 4 from the nodesource repo for Debian/jessie and Ubuntu/trusty, but I'd like to avoid it.